bot/cogs/commands.py

```diff
@@ -470,7 +634,10 @@ async def remove_dialogflow(self, ctx, intent_id: str, *args):
         await self.bot.reply_to_msg(ctx.message, "Dialogflow reply deleted")
 
     @add.command(name="dialogflowChannel")
-    async def add_dialogflow_channel(self, ctx, channel: commands.TextChannelConverter):
+    async def add_dialogflow_channel(self, ctx: commands.Context, channel: commands.TextChannelConverter):
+        """Usage: `add dialogflowChannel (channel)`
+        Purpose: Adds channel to the list of channels that natural language processing is applied to
+        Notes: probably don't mess around with this, Mircea is the only wizard that knows how these works"""
```

Comment:
This is just a command to add a channel to a list, no need for that note :smile: (same for the other commands below)